documentation/Triggers-and-Actions.md

```diff
@@ -4,14 +4,15 @@
 
 ## Article Contents
 
-- [Overview of Trigger and Action system](#overview-of-trigger-and-action-system)
-  - [List of available Triggers](#list-of-available-triggers)
-  - [Database tables that have a TRIGGER field](#database-tables-that-have-a-trigger-field-that-is-listened-to)
-  - [List of Core-actions available in initial configuration](https://github.com/openmsupply/application-manager-web-app/wiki/List-Core-Actions)
-- [Actions](#actions)
-- [Action parameters](#action-parameters)
-- [Passing information to Actions](#passing-information-to-actions)
-- [Running actions via REST endpoint](#running-actions-via-rest-endpoint)
+- [Triggers & Actions](#triggers--actions)
```

Comment:
```suggestion
- [Triggers & Actions](#triggers-actions)
```

Reply:
This is auto-generated and it's technically correct since the link it jumps to has an `&` in the middle.

But it's weird to have a link to the top of the page that we're already at, so I've just removed it altogether and indented the TOC back one. e2e62e25